app/presenters/vacancy_presenter.rb

```diff
@@ -108,4 +108,8 @@ def show_job_roles
   def show_subjects
     model.subjects&.join(', ')
   end
+
+  def immediate_parent_organisation_type
```

Comment:
My concern with adding this code is two-fold:

1. We're moving away from the use of presenters
2. This tabs design is being replaced imminently with a pills design

My proposal would be either:

- Leave the small piece of logic for the tab heading as it was, or
- Use a small inline view_component 

I would lean towards the first option since this will be refactored soon anyway, and this isn't being done as part of any ticket